### PR TITLE
Setup storyshots

### PR DESCRIPTION
**Pull request by aalmacin**

### CODE REVIEW

File: lib/button/button.component.spec.ts [hunk 1]
```diff
@@ -1,24 +1,49 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { async, ComponentFixture, TestBed } from "@angular/core/testing";
 
-import { ButtonComponent } from './button.component';
+import { ButtonComponent } from "./button.component";
 
-describe('LibComponent', () => {
-  let component: ButtonComponent;
-  let fixture: ComponentFixture<ButtonComponent>;
+import * as puppeteer from "puppeteer";
```
Comment (johannalee) [inline lib/button/button.component.spec.ts]: so if we are using puppeteer then we don't need to spike on testcafe

File: package.json [hunk 2]
```diff
@@ -10,16 +10,18 @@
...
+    "test:update-snapshot": "export NODE_ENV=test&& ng test --updateSnapshot",
     "lint": "ng lint",
     "lint:fix": "ng lint --fix",
     "format": "prettier --config .prettierrc --list-different \"./lib/**/*.{ts,js,scss}\"",
     "format:fix": "prettier --config .prettierrc --write \"./lib/**/*.{ts,js,scss}\""
   },
   "husky": {
     "hooks": {
-      "pre-commit": "ng test && npm run format"
+      "pre-commit": "export NODE_ENV=test&& ng test && npm run format"
```
Comment (johannalee) [inline package.json]: spacing around `&&`

File: lib/button/button.component.spec.ts [hunk 3]
```diff
@@ -1,24 +1,49 @@
...
 
-  beforeEach(async(() => {
-    TestBed.configureTestingModule({
-      declarations: [ButtonComponent]
-    }).compileComponents();
-  }));
+describe("LibComponent", () => {
```
Comment (johannalee) [inline lib/button/button.component.spec.ts]: should be ButtonComponent

File: lib/button/button.component.spec.ts [hunk 4]
```diff
@@ -1,24 +1,49 @@
...
+  describe("DOM", () => {
```
Comment (johannalee) [inline lib/button/button.component.spec.ts]: i would elaborate more `DOM Snapshot`

File: lib/button/button.component.spec.ts [hunk 5]
```diff
@@ -1,24 +1,49 @@
...
-  it('should create', () => {
-    expect(component).toBeTruthy();
+  describe("Image Snapshot", () => {
+    let browser;
+
+    beforeEach(async () => {
+      browser = await puppeteer.launch();
+    });
+
+    it("default", async () => {
```
Comment (johannalee) [inline lib/button/button.component.spec.ts]: `it should take a default image shot`

**Review by johannalee — COMMENTED** [review]
again a few small comments

File: lib/button/button.component.spec.ts (hunk 1 above)
Comment (aalmacin) [reply lib/button/button.component.spec.ts]: We'll still use TestCafe for E2E

**Review by johannalee — COMMENTED** [review]
:shipit: but Keep in mind that we will have to replace puppeteer with the to-be-chosen e2e framework or just keep it.

**Review by johannalee — APPROVED** [review]

**Pull request closed without merging**